lib/pymedphys/_mosaiq/api.py

```diff
@@ -113,6 +131,8 @@ def execute(
     ...     ''',
     ...     {"last_name": "PHANTOM"},
     ... )  # doctest: +SKIP
+
+    >>> list(results)  # doctest: +SKIP
```

Comment:
What are your thoughts about changing the execute results to a generator? Users are free to wrap the result in `list` to get what they're used to, but it means that if one is to iterate over a table one only needs to retrieve one item at a time, instead of having to pull the full table from Mosaiq all at once...